_aicspylibczi/SourceRange.h

```diff
@@ -59,7 +59,7 @@ namespace pylibczi {
               return !(*this==other_);
           }
 
-          std::vector<T*> operator*()
+          std::vector<T*> &operator*()
```

Comment:
This suddenly looks odd to me.  a variable called `m_ptr` is actually a vector of pointers?

Reply:
so you want a better name? m_vPtr or m_ptrVec ?

Reply:
Yes -- how about m_channelPtrs or even m_channels?